gst/tensor_decoder/tensordec.c

```diff
@@ -96,9 +97,30 @@ enum
 {
   PROP_0,
   PROP_OUTPUT_TYPE,
-  PROP_SILENT
+  PROP_SILENT,
+  PROP_LABEL
```

Comment:
Labeling is only one of the possible options. Thus, I'd suggest:
```
PROP_MODE,
PROP_MODE_OPTION1,
PROP_MODE_OPTION2,
```

Then, ```MODE``` = ```{ "direct_video", "image_labeling" }``` at this version.
With ```direct_video``` mode, ```MODE_OPTION*``` is ignored.
With ```image_labeling" mode, ```MODE_OPTION1``` is the file path to the label file.

THen, with the same property variables, we can support other "modes" later without adding more property variables.